chord_metadata_service/chord/ingest.py

```diff
@@ -62,6 +63,27 @@
                     "value": "{json_document}"
                 }
             ]
+        },
+        "fhir_json": {
+            "name": "FHIR resources json",
+            "description": "This ingestion workflow will validate and import a FHIR schema-compatible "
+                           "JSON document.",
+            "data_type": "experiment",
```

Comment:
data_type should either be fhir or phenopacket (probably latter here, see above)

Reply:
I will fix to for 'phenopacket'.